core/src/main/java/org/aerogear/mobile/core/metrics/publisher/NetworkMetricsPublisher.java

```diff
@@ -54,12 +54,10 @@ public void publish(final Metrics... metrics) {
 
             final HttpResponse httpResponse = httpRequest.execute();
             httpResponse.onComplete(() -> {
-                if (httpResponse.getStatus() == HTTP_OK) {
-                    MobileCore.getLogger().debug("Metrics sent: " + json.toString());
-                } else {
-                    MobileCore.getLogger().error(httpResponse.getRequestError().getMessage(),
-                        httpResponse.getRequestError());
-                }
+                MobileCore.getLogger().debug("Metrics sent: " + json.toString());
+            }).onError(() -> {
+                MobileCore.getLogger().error("Metrics request error",
```

Comment:
@secondsun @danielpassos @wtrocki Should we log anything here? This will happen on devices out in the field, while the logs are only useful when plugged into Android Studio. Are you aware of any conventions about logging in Android apps?

Reply:
Great to see this example aligned with IOS:
https://github.com/aerogear/aerogear-ios-sdk/blob/master/modules/core/metrics/engine/MetricsNetworkPublisher.swift#L20-L26

This can be ok for now. For the cases when network connection is down error will be printed but that's not a big problem IMHO.
I have no strong opinion about it, but would like to keep that aligned with IOS.

Reply:
Actually this area will need more work anyway.